packit_service/worker/mixin.py

```diff
@@ -375,17 +377,15 @@ def identifier(self) -> str:
 
     @property
     def owner(self) -> str:
-        return self.job_config.owner
+        return self.job_config.owner or self.copr_build.owner
```

Comment:
Since the `copr_build` can be `None` and also the `job_config.owner` should we manage it in a different way? Or do you think it will not happen that both of them are `None`?

Reply:
good point! fixed